plugins/Chialab/src/Controller/AppController.php

```diff
@@ -78,14 +80,20 @@ public function beforeRender(EventInterface $event): void
         parent::beforeRender($event);
 
         $root = $this->Publication->getPublication();
-        $menu = $this->Menu->load((string)$root->id);
+        $menu = Cache::remember('menu', function () use ($root) {
+            return $this->Menu->load((string)$root->id);
+        });
```

Comment:
Same, but with arrow functions:

```suggestion
        $menu = Cache::remember(
            'menu',
            fn () => $this->Menu->load((string)$root->id),
        );
```